packages/iov-tendermint-rpc/src/client.ts

```diff
@@ -89,6 +91,24 @@ export class Client {
     return this.doCall(query, this.p.encodeStatus, this.r.decodeStatus);
   }
 
+  public subscribe(
+    eventType: SubscriptionEventType,
+    query?: SubscribeRequestQuery,
+  ): Stream<responses.SubscriptionEvent> {
+    if (!instanceOfRpcStreamingClient(this.client)) {
+      throw new Error("This RPC client type cannot subscribe to events");
+    }
+
+    const request: requests.SubscribeRequest = { method: Method.SUBSCRIBE, type: eventType };
+    const req = this.p.encodeSubscribe(request, query || undefined);
+    const eventStream = this.client.listen(req);
+    return eventStream.map<responses.SubscriptionEvent>(event => {
+      // tslint:disable-next-line:no-console
+      // console.log(JSON.stringify(event));
```

Comment:
uncomment me for debugging

Reply:
This was a very useful comment. Can probably go now that it all works with #298, right?